keras2onnx/ke2onnx/gru.py

```diff
@@ -85,13 +85,14 @@ def convert_keras_gru(scope, operator, container):
         op_version = 1
         attrs['output_sequence'] = 1 if output_seq else 0
     elif container.target_opset <= 5:
-        attrs['linear_before_reset'] = 0
         attrs['output_sequence'] = 1 if output_seq else 0
         op_version = 3
     else:
-        attrs['linear_before_reset'] = 0
         op_version = 7
 
+    if container.target_opset >= 3:
+        attrs['linear_before_reset'] = 1 if op.reset_after else 0
```

Comment:
No update for bias weights?

Reply:
No need, The bias tensor for the gates: Concatenation of `[Wb[zrh], Rb[zrh]]` is passed in.